README.md

```diff
@@ -19,22 +19,31 @@ You can either use Donut directly or check out the available plugins:
 ## Release Notes
 See what's new [here](release-notes.md)
 
-### download
+### Download
 ```
 wget http://repo1.maven.org/maven2/io/magentys/donut/1.0/donut-1.0-one-jar.jar
 ```
 or download the latest release from: [here](http://repo1.maven.org/maven2/io/magentys/donut/1.0/donut-1.0-one-jar.jar)
 
-### run from command line
+### Run from command line
 
 ```
 java -jar donut-<Version>.jar -s cucumber:/my/path/cucumber-reports -n myProjectName
 ```
 
-### options
+### Run using Docker
+
+Donut is packaged as a [Docker container](https://hub.docker.com/r/donutreport/donut-report/)
+
+To run using Docker:

```

Comment:
You added a clarifying statement here regarding the version.  By default, the current command will choose the `LATEST` image available.

I've extracted the version into an ENV.  This will need to be updated as part of the release, etc.
I've updated the instructions for tagging/deploy to specify the version number.
The version of the  container, should match the version of Donut Maven project..